src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -5,68 +5,86 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
-using AutoRest.CSharp.V3.ClientModels.Serialization;
-using AutoRest.CSharp.V3.Pipeline.Generated;
-using AutoRest.CSharp.V3.Plugins;
+using AutoRest.CSharp.V3.Input;
+using AutoRest.CSharp.V3.Input.Generated;
+using AutoRest.CSharp.V3.Output.Models;
+using AutoRest.CSharp.V3.Output.Models.Requests;
+using AutoRest.CSharp.V3.Output.Models.Responses;
+using AutoRest.CSharp.V3.Output.Models.Serialization;
+using AutoRest.CSharp.V3.Output.Models.Shared;
+using AutoRest.CSharp.V3.Output.Models.TypeReferences;
 using AutoRest.CSharp.V3.Utilities;
 using Azure.Core;
-using HttpHeader = AutoRest.CSharp.V3.Pipeline.Generated.HttpHeader;
+using HttpHeader = AutoRest.CSharp.V3.Input.Generated.HttpHeader;
```

Comment:
Why is this required? There should be no name conflicts.

Reply:
![image](https://user-images.githubusercontent.com/17788297/72466171-3954da00-378d-11ea-9e91-85626b85eb69.png)

Reply:
Sure, rename one of those types.

Reply:
You really want me to rename the type from the CodeModel? I didn't change any CodeModel types in this PR.

Reply:
Done